### ISSUE TITLE
[Feature Request] Hide Passkeys information by default

### ISSUE THREAD

**Issue by DannieBGoode** [strongbox-password-safe/Strongbox#821]
## Problem
Creating a passkey in Strongbox generates a series of fields within the KeePass entry. These fields do not have much value when accessing the login entry since the user very rarely (if never) will be copy pasting those values anywhere. This creates **a very crowded details page** within the entry that adds a lot of noise and makes finding the right information more difficult.

### Easy Solution
I suggest the Passkey details are hidden behind a toggle the same way that the _Metadata_ or the _History_ hides information which is rarely accessed behind them.

### Medium Solution
ProtonPass has a great implementation when displaying passkeys that is very attractive. I suggest looking into it.

**Comment by strongbox-mark** [comment]
Hi @DannieBGoode - Thanks, could you attach a sample ProtonPass screenshot?

**Comment by DannieBGoode** [comment]
> Hi @DannieBGoode - Thanks, could you attach a sample ProtonPass screenshot?

In the entry you can see there is a field called "Passkey" with the email of the account associated to that passkey. That entry is unlike the others because it is clickable.


![image](https://github.com/user-attachments/assets/b5cf4b57-f7f5-4fdd-8c63-d97ae5a934dc)

If you click on it you get a popup with more details on the passkey.

![image](https://github.com/user-attachments/assets/c54424b0-0438-425d-a1db-035a43f9a2b1)

The screenshots are taken on Desktop but it works exactly the same way on iOS.

**Comment by strongbox-mark** [comment]
Yes, looks better, I tend to agree we should probably collapse most of those details.
